### PR TITLE
[Doc] Add steps to remove weave-net

### PR DESCRIPTION
**Pull request by Binyang2014**

### CODE REVIEW

**Comment by ydye** [comment]
Is it possible to write an ansible role to do this work?

**Comment by Binyang2014** [comment]
> Is it possible to write an ansible role to do this work?

I am not sure. The `kubectl` need to run on dev-box, Seems we need to add `localhost` in the `inventory`.
The `kubectl` command only need to run once. It's not time consuming. And admin should have basic knowledge about k8s.

**Comment by ydye** [comment]
> > Is it possible to write an ansible role to do this work?
> 
> I am not sure. The `kubectl` need to run on dev-box, Seems we need to add `localhost` in the `inventory`.
> The `kubectl` command only need to run once. It's not time consuming. And admin should have basic knowledge about k8s.

Agree with you about the point that admin should have knowledge of k8s.

File: docs/manual/cluster-admin/installation-faqs-and-troubleshooting.md [hunk 1]
```diff
@@ -60,18 +60,38 @@ To remove the network plugin, you could use following `ansible-playbook`:
...
+    
+    - name: restart docker
+      shell: systemctl restart docker
       args:
         executable: /bin/bash
 ```
 
-After this step, if your pod still can not access internet, please change the pod spec to use `hostNetwork`.
+After this step you need to change the `coredns` to fix dns resolution issue.
+Please run `kubectl edit cm coredns -n kube-system -o yaml`, change `.:53 {` to `.:9053`
```
Comment (abuccts) [inline docs/manual/cluster-admin/installation-faqs-and-troubleshooting.md:92]: what is ` {`

Comment (Binyang2014) [reply docs/manual/cluster-admin/installation-faqs-and-troubleshooting.md]: Thanks, it's typo. Fixed

**Pull request merged**